hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueContainerMetadataInspector.java

```diff
@@ -525,12 +524,12 @@ static PendingDelete countPendingDeletesSchemaV2(
 
   static long computePendingDeleteBytes(List<Long> localIDs,
       KeyValueContainerData containerData,
-      Table<String, BlockData> blockDataTable) {
+      DatanodeStoreWithIncrementalChunkList schemaTwoStore) {
```

Comment:
It looks like both countPendingDeletesSchemaV3 and countPendingDeletesSchemaV2 will call this computePendingDeleteBytes.  Could you rename schemaTwoStore to maybe only "store"?